pystarport/pystarport/default.nix

```diff
@@ -0,0 +1,12 @@
+{ pkgs ? import ../../nix { }, commit ? "" }:
```

Comment:
- this is the directory of python package, better to put config files outside
- maybe we make the proto generated files as a nix package, which can be depend by pystarport package?

Reply:
i'll try not to use ../../nix